README.md

```diff
@@ -41,21 +44,25 @@ git clone https://github.com/usdot-its-jpo-data-portal/canary-lambda.git
 3. Set the Execution role to one that has the S3 and SES permissions listed in the **Requirements** section above.
 4. Set the **Memory (MB)** to `512 MB` and the **Timeout** to `1 min 0 sec`.
 ![Lambda Settings](images/figure2.png "Lambda Settings")
+5. Set the environment variables.
+![Environment Variables](images/figure3.png "Environment Variables")
 
 ## Configuration
 
-**Note: This configuration is still in progress and will be added in the future.**
+Some of these variables are general settings that may be changed in the code (Source = Code). To change these, edit them either locally and reupload the file, or change them in the Lambda console editor.
```

Comment:
Why having separate ways? Any problem with doing all the variables through environment?